### ISSUE TITLE
verbose parameter during training should probably be 1 not 2

### ISSUE THREAD

**Issue by 459below** [rodrigo2019/keras-yolo2#26]
https://github.com/rodrigo2019/keras_yolo2/blob/3ed7a365ceb2b8e473e0865cd7e4023325b5c360/keras_yolov2/frontend.py#L216

To my knowledge the verbose parameter in Keras is pretty confusing since it does not behave as one is used to. Higher doesn't seem to mean "more verbose" in this case. If debugging I'm quite sure that it should be verbose=1 instead of verbose=2.

According to [documentation](https://www.tensorflow.org/api_docs/python/tf/keras/Model)

verbose | 0, 1, or 2. Verbosity mode. 0 = silent, 1 = progress bar, 2 = one line per epoch. Note that the progress bar is not particularly useful when logged to a file, so verbose=2 is recommended when not running interactively (eg, in a production environment).

**Comment by rodrigo2019** [comment]
it is because in the older versions of the repo, we aplied some tf.print() lines in the loss function method. 
I will fix that, thank you
